vnext/Microsoft.ReactNative/QuirkSettings.idl

```diff
@@ -26,5 +26,12 @@ namespace Microsoft.ReactNative
 
     DOC_STRING("Runtime setting allowing Networking (HTTP, WebSocket) connections to skip certificate validation.")
     static void SetAcceptSelfSigned(ReactInstanceSettings settings, Boolean value);
+
+    DOC_STRING(
+      "Runtime setting that can be used to skip the default back handler logic.  "
+      "If you have an application that wants to handle back bahavior in native code "
+      "instead of JavaScript, setting this to false will prevent the native module "
```

Comment:
Am I reading this correct?
Statement: `wants to handle ... in native instead of JavaScript`
My interpretation: This leads me to believe the default is JavaScript. 

Statement: `Default: true`
My interpretation: If default is `true`, then changing the setting to `false` should prevent JavaScript from handling and Native to handle it.

Statement: `Setting this to false will prevent native from handling the events`
My confusion: I read this as being in conflict with my two interpretations above :)

Reply:
yeah I am equally confused by the wording here. Is this supposed to say "if you want to handle back behavior in JavaScript instead of native, set this to true"?
I feel like this API would be better served by an enum:

```cs
enum BackNavigationHandlerKind {
  JavaScript = 0,
  Native = 1
}

static void SetBackHandler(ReactInstanceSettings settings, BackNavigationHandlerKind kind);
```